docker-compose.yml

```diff
@@ -38,4 +38,16 @@ services:
   redis:
     image: redis
     ports:
-      - 6379:6379
\ No newline at end of file
+      - 6379:6379
+
+  elasticsearch:
```

Comment:
it needs to be added to the `start-dependencies.sh`, if not the pipeline won't pass.
`docker-compose -f docker-compose.yml up -d minio localstack redis mongo elasticsearch`

Reply:
althought we could also run all the docker-compose services without need of specifying each of them, so in the future we'll avoid new contributions to worry about this .
it would just be `docker-compose -f docker-compose.yml up -d`